build/__tools__/verify.sh

```diff
@@ -112,7 +112,7 @@ fi
 if [ "${SGX_MODE}" = "HW" ]; then
     # IAS KEY should be a PEM file ..
     if [ ! -f "${PDO_IAS_KEY_PEM}" ]; then
-        warn "PDO_IAS_KEY_PEM '${PDO_IAS_KEY_PEM}' file does not exist"
+        recho "PDO_IAS_KEY_PEM '${PDO_IAS_KEY_PEM}' file does not exist, will be built"
```

Comment:
Because `warn` actually raises an error and the verification fails.
However, as we are now using a default value, I might as well remove this completely.

Reply:
I have removed variable and check